cmd/backup/main.go

```diff
@@ -65,4 +82,83 @@ func main() {
 	s.must(s.withLabeledCommands(lifecyclePhaseProcess, s.encryptArchive)())
 	s.must(s.withLabeledCommands(lifecyclePhaseCopy, s.copyArchive)())
 	s.must(s.withLabeledCommands(lifecyclePhasePrune, s.pruneBackups)())
+
+	return nil
+}
+
+func main() {
+	serve := flag.Bool("foreground", false, "run the backup in the foreground")
```

Comment:
```suggestion
	serve := flag.Bool("foreground", false, "run the tool in the foreground")
```

"running the backup" is a bit ambiguous.

Reply:
Done